python/mdns.py

```
@@ -0,0 +1,163 @@
+        #for _ in range(qdcount):
+        #    _, offset = parse_dns_name(data, offset)
+        #    offset += 4
+
+        answers = arcount + ancount
+        log.debug("Response received: {} answer(s)".format(answers))
+
+        # Parse Answers
+        for _ in range(answers):
+            name, offset = parse_dns_name(data, offset)
+            rtype = struct.unpack(">H", data[offset: offset+2])[0]
+            rclass = struct.unpack(">H", data[offset+2: offset+4])[0]
```

Ruff (0.12.2)

Local variable `rclass` is assigned to but never used

Remove assignment to unused variable `rclass`

(F841)